src/domain/type.ts

```diff
@@ -0,0 +1,17 @@
+export type CardNumbersType = {
+  first?: string;
+  second?: string;
+  third?: string;
+  fourth?: string;
+};
```

Comment:
유틸리티 타입을 사용하시면 수고로움을 덜 수 있답니다!

```suggestion
export type CardNumbersType = Partial<{
  first: string;
  second: string;
  third: string;
  fourth: string;
}>;
```